techniques/applications/aptPackageManagerSettings/3.0/metadata.xml

```diff
@@ -23,6 +23,7 @@ appreciated by administrators on the APT package
 manager, and eventually enforce them.
 -->
 <TECHNIQUE name="APT package manager configuration">
+  <DEPRECATED>This technique version has been superseded by a new version. It will no longer be available in the next stable version of Rudder. Please upgrade to the latest version.</DEPRECATED>
```

Comment:
are you sure you already want to deprecate it ?
shouldn't it be deprecated in 3.3 ?